po/ca.po

```diff
@@ -156,358 +191,372 @@ msgid ""
 msgid "Fixed Twitch livestreams"
-msgstr ""
+msgstr "S'han corregit les transmissions en directe de Twitch"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:114
 msgid "Fixed YouTube resolution support: now up to 4K!"
-msgstr ""
+msgstr "S'ha corregit la compatibilitat amb les resolucions de YouTube: ara fins 4K!"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:115
 msgid "Code cleaning to keep things fresh and clean"
-msgstr ""
+msgstr "Neteja de codi per mantindre les coses netes i ordenades"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:121
 msgid "Better error handling and quality of life improvements all around:"
-msgstr ""
+msgstr "Millor control d'errors i millores de la qualitat a tots els llocs:"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:123
 msgid "New approval dialogs for external apps, e.g. appstream:// or ftp://"
-msgstr ""
+msgstr "Nous diàlegs de confirmació per aplicacions externes, p.e. appstream:// o ftp://"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:124
 msgid "New warning when no network connection"
-msgstr ""
+msgstr "Nou avís quan no hi ha connexió a la xarxa"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:125
 msgid "Prettier errors using native views"
-msgstr ""
+msgstr "Errors més bonics utilitzant les vistes natives"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:126
 msgid "Stopped throwing away text when unfocusing URL entry"
-msgstr ""
+msgstr "S'ha deixat de llançar text quan l'entrada d'URL perd el focus"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:127
 msgid "Stopped searching when the URL entry is empty"
-msgstr ""
+msgstr "S'ha deixat de cercar quan l'entrada d'URL està buida"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:128
 msgid "Code cleaning to keep things smelling fresh"
-msgstr ""
+msgstr "Neteja de codi per mantindre les coses amb olor a fresc"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:134
 msgid "Fresh styles to keep things funky, plus more:"
-msgstr ""
+msgstr "Nous estils per mantindre les coses a la moda, i a més:"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:136
 msgid ""
 "The new start page makes Ephemeral faster and makes it not touch the network "
 "until you're ready to browse"
 msgstr ""
+"La nova pàgina d'inici fa Ephemeral més ràpid i que no toque la xarxa "
+"fins que estigueu preparats per navegar"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:137
 msgid "Middle- or Ctrl-click a link to open it in a new window"
-msgstr ""
+msgstr "Féu Ctrl-click o click amb el botó del mig per obrir un enllaç a una nova finestra"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:138
 msgid ""
 "Displayed URLs are kept in sync for pages that manually update the URL "
 "without a page load"
 msgstr ""
+"Les URLs mostrades es mantenen sincronitzades per aquelles pàgines que actualizen manualment "
```

Comment:
_URL_ is masculine (_Llocalitza**dor** de recurs uniform_) and takes no plural _s_ as any other acronym: “_Els URL mostrats…_”